go/worker/compute/executor/committee/txnscheduling/algorithm/batching/batching.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
 	"github.com/oasisprotocol/oasis-core/go/common/logging"
 	registry "github.com/oasisprotocol/oasis-core/go/registry/api"
-	"github.com/oasisprotocol/oasis-core/go/worker/compute/txnscheduler/algorithm/api"
+	"github.com/oasisprotocol/oasis-core/go/worker/compute/executor/committee/txnscheduling/algorithm/api"
```

Comment:
Maybe consider moving the `algorithm` package to `go/runtime/scheduling` (and rename to `scheduling`) as it seems quite independent from the actual worker.